econml/utilities.py

```diff
@@ -1266,6 +1266,18 @@ def transpose_dictionary(d):
     return output
 
 
+def reshape_arrays_2dim(length, *args):
+    new_args = []
+    for arg in args:
+        if arg is None:
+            new_args.append(np.array([]).reshape(length, 0))
```

Comment:
Is this logic correct?  For example, if X is None for DML, then that's not equivalent to passing a 0-column X, it's equivalent to passing a column of ones for X instead.  I would expect that the right thing to do is to just pass None through to the output and let the estimator handle it accordingly.

Reply:
This function is only used to concatenate a pandas dataframe as the input to dowhy CausalModel(). I don't want to check one by one whether they are None, when concatenate data and column names.

Reply:
Hmm... So does X=None work for DML through dowhy?